src/sagemaker/amazon/amazon_estimator.py

```diff
@@ -622,13 +637,25 @@ def get_image_uri(region_name, repo_name, repo_version=1):
         if supported_version:
             return get_xgboost_image_uri(region_name, supported_version[0])
 
-        logging.warning(
-            "There is a more up to date SageMaker XGBoost image. "
-            "To use the newer image, please set 'repo_version'="
-            "'%s'. For example:\n"
-            "\tget_image_uri(region, 'xgboost', '%s').",
-            XGBOOST_VERSION_1,
-            XGBOOST_VERSION_1,
-        )
     repo = "{}:{}".format(repo_name, repo_version)
     return "{}/{}".format(registry(region_name, repo_name), repo)
+
+
+def _is_latest_xgboost_version(repo_version):
+    """Compare xgboost image version with latest version
+
+    Args:
+        repo_version:
+    """
+    if repo_version == 1 or repo_version == 'latest':
+        return True
+    return _split(XGBOOST_LATEST_VERSION) <= _split(repo_version)
```

Comment:
I forgot about this earlier, but we should also consider `{XGBOOST_LATEST_VERSION}-cpu-py3` as valid for the latest version. Let's make sure this value for `repo_version` returns true as well.

Also, let's use `==` instead of `<=`. `<=` would allow invalid versions to return `True` if they are "larger" than `XGBOOST_LATEST_VERSION`, which doesn't match what we're trying to do here.